client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/restaurant/RestaurantScreenModel.kt

```diff
@@ -313,7 +310,17 @@ class RestaurantScreenModel(
     }
 
     private fun onCreateCuisinesSuccessfully(cuisineName: String?) {
-        getCuisines()
+        updateState {
+            it.copy(
+                restaurantAddCuisineDialogUiState = it.restaurantAddCuisineDialogUiState.copy(
+                    isVisible = true,
+                    cuisineName = "",
+                    cuisines = it.restaurantAddCuisineDialogUiState.cuisines.toMutableList().apply {
+                        add(cuisineName!!)
```

Comment:
if we are sure the cuisine name is not nullable then why the function accept a nullable string ? so let's the function accept only a non nullable string